local-volume/provisioner/pkg/common/common.go

```diff
@@ -122,6 +126,26 @@ func CreateLocalPVSpec(config *LocalPVConfig) *v1.PersistentVolume {
 	}
 }
 
+// GetVolumeConfigFromMountedConfigMap gets volume configuration from the mounted configmap volume,
+func GetVolumeConfigFromMountedConfigMap() (map[string]MountConfig, error) {
+	volumes := make(map[string]string)
+	fullConfigPath := path.Join(ProvisionerConfigPath, "..data/")
```

Comment:
Why do we need the backstep?

Reply:
it is the format of path when configmap is mounted:

/etc/provisioner/config/..data/blah
there is no backstep, it is just how they name "..data", not sure where it comes from though, but it is like that.

Reply:
Hmm that is different than what they have documented.  Can you check with api machinery team?

Reply:
I will but I have working proof and in my other project kolla-kubernetes we saw exactly the same behavior when we needed to access mounted configmap.

Reply:
Is there a particular concern? I am not sure I understand the problem.

Reply:
No, just that it seems like it conflicts with the documentation, so either the documentation needs to be updated, or we're missing something?

Reply:
..data is an implementation detail of kubernetes "atomic writer" for writing/updating configmap, secret volumes and the like. this issue is irrelevant but has a nice `tree` output so see here: https://github.com/kubernetes/kubernetes/issues/50345#issuecomment-321346592

Reply:
in short, if possible this should rely on the symlink `blah` instead of the file `..data/blah` if possible

Reply:
Thanks for the explanation!  Yes, let's not use ..data

Reply:
By some reason when I tried last time to use sylink it did not work in some cases and using ..data proved to be the most reliable way. But if you are not afraid of the future headache, I can switch to using symlink.

Reply:
If there's a problem using the documented way, then it's a bug that needs to be fixed.

Reply:
@msau42 could you please point me to the documentation which show how to access files in the container by using symlink directory?

Reply:
I was just looking at the configmap doc here: https://kubernetes.io/docs/tasks/configure-pod-container/configure-pod-configmap/#add-configmap-data-to-a-volume

From their example, it looks like all the directories should be accessible right at the mount path.